jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenter.java

```diff
@@ -241,5 +245,7 @@ public IsWidget getProcessDiagramView() {
         void displayAllTabs();
 
         void displayOnlyLogTab();
+
+        void displayNotification(String text);
```

Comment:
@nmirasch @bsremac  not sure if you agree but I've mixed feelings about this display notification in the view classes, because it is an event, we could consider that it could leave in the presenter class, wdyt?

Reply:
@cristianonicolai @nmirasch Now when mentioned, I agree it makes more sense to handle it inside the presenter